packages/components/navigation/Sidebar.tsx

```diff
@@ -124,7 +124,13 @@ export const Sidebar: React.FC = () => {
           return (
             <SidebarButton
               key={item.id}
-              onPress={onRouteChange}
+              onPress={
+                route === "External"
```

Comment:
Maybe you can type `route` from `getFromFile.ts` with something like `"External" | "ComingSoon" | keyof RootStackParamList`

Reply:
thanks for the code; 
Updated!